app/views/graphs/index.haml

```diff
@@ -11,17 +11,22 @@
 
 .card
   .card-header
-    %h3 The current per head values for garabe
+    %h3= "Monthly Statistics: #{@date.strftime("%B %Y")}"
   .card-body
     %table.table
       %thead
         %tr
-          %th Kind
-          %th per head/per month
+          %th.text-left Kind
+          %th.text-left Your household (total)
+          %th.text-left Your household (per person)
+          %th.text-right Average household (per person)
           %th Source
       %tbody
         - GraphPresenter::GARBAGE.each do |key, value|
+          - household_weight = Pile.where(user_id: current_user.id).where(kind: key).group("DATE_TRUNC('month', produced_at)").sum(:weight).values.first
```

Comment:
This should go to a presenter, but anywise my plan was also to refactor the whole logic, so maybe currently i see this as a optimization.